src/bundle/Resources/views/themes/admin/content/edit/base.html.twig

```diff
@@ -23,21 +23,60 @@
                 {{ form_start(form, {'attr': {'class': 'ez-form-validate'}}) }}
 
                 {% block form_fields %}
-                    {% for field in form.fieldsData %}
-                        {% if not field.rendered %}
-                            {% if field.value is defined %}
-                                {{- form_widget(field) -}}
-                            {% else %}
-                                <div>
-                                    {{- form_label(field) -}}
-                                    <p class="non-editable">
-                                        {{ "content.field.non_editable"|trans|desc('This Field Type is not editable') }}
-                                    </p>
-                                    {% do field.setRendered() %}
+                    {% if grouped_form_fields_views is defined and grouped_form_fields_views|length > 1 %}
```

Comment:
Please update the name `grouped_form_fields_views` to `grouped_fields` after changes in https://github.com/ezsystems/ezplatform-content-forms/pull/26/files